docs/modules/ROOT/pages/subscriptions/events/connect.adoc

```diff
@@ -0,0 +1,698 @@
+[[connect]]
+= Connect Subscriptions
```

Comment:
Maybe this should be `CreateRelationship`  instead of `Connect`?

Reply:
Referencing the PR for this here #2475

Reply:
@a-alle since that PR has been merged, does this document need to be updated to reflect the name change?